mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -24,8 +24,8 @@ import scala.collection.mutable.ListBuffer
 import breeze.linalg.{axpy => brzAxpy, inv, svd => brzSvd, DenseMatrix => BDM, DenseVector => BDV,
   MatrixSingularException, SparseVector => BSV}
 import breeze.numerics.{sqrt => brzSqrt}
-
 import org.apache.spark.annotation.Since
+import org.apache.spark.internal.config.MAX_RESULT_SIZE
```

Comment:
The style checker is picky:
```
[error] /home/jenkins/workspace/NewSparkPullRequestBuilder/mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala:27:0: There should at least one a single empty line separating groups 3rdParty and spark.
[error] /home/jenkins/workspace/NewSparkPullRequestBuilder/mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala:29:0: org.apache.spark.internal.Logging is in wrong order relative to org.apache.spark.internal.config.MAX_RESULT_SIZE.
[error] Total time: 18 s, completed Mar 11, 2019 6:05:24 AM
```

I think you need to restore the blank line, and move this import below the following one.

Reply:
I'll try that.
However, I don't understand the rationale behind this import ordering error, as "org.apache.spark.internal.Logging" seems to be higher in alphabetic order than "org.apache.spark.internal.config.MAX_RESULT_SIZE"

Reply:
It's sorted by package, then within the package. org.apache.spark.internal is considered to sort before subpackages like org.apache.spark.internal.config

Reply:
okay makes sense, thanks for the clarification !